xpk/xpk.py

```diff
@@ -473,6 +473,14 @@ def add_env_config(args):
               'environment, a value must be specified.'
           )
           env[variable] = os.environ[variable]
+
+  if args.debug_dump_gcs:
+    if 'XLA_FLAGS' in env:
```

Comment:
Shouldn't we check if `xla_dump_to` is in the XLA_FLAGS? That way the user can set `XLA_FLAGS` (for other things).

Reply:
+1 we can support more adding `xla_dump_to` the XLA_FLAGS if `xla_dump_to` is not specified by the env file.